cmake/genmsg-extras.cmake.em

```diff
@@ -18,6 +18,9 @@ set(GENMSG_CHECK_DEPS_SCRIPT "${genmsg_DIR}/../../../@(CATKIN_PACKAGE_BIN_DESTIN
 
 include(CMakeParseArguments)
 
+# We need to call assert_file_exists()
```

Comment:
Please either remove the comment (since the mentioned function is not the only thing used from catkin) or extend to all parts of catkin which are used. Since the second option is likely getting outdated I would suggest the first one.